README.md

```diff
@@ -38,72 +38,125 @@ With a Flow pipeline, you:
       - resource:
-          stream: tripdata/
+          stream: tripdata/JC-201703
           syncMode: incremental
-        target: acmeCo/tripdata
-        
-materializations: 
-  acmeCo/postgres: 
-    bindings: 
-      - source: acmeCo/tripdata 
-        resource: 
-          table: trips 
+        target: examples/citi-bike/rides
+
+collections:
+  # A collection of Citi Bike trips.
+  examples/citi-bike/rides:
+    key: [/bike_id, /begin/timestamp]
+    # JSON schema against which all trips must validate.
+    schema: https://raw.githubusercontent.com/estuary/flow/master/examples/citi-bike/ride.schema.yaml
+    # Projections relate a tabular structure (like SQL, or the CSV in the "tripdata" bucket)
+    # with a hierarchical document like JSON. Here we define projections for the various
+    # column headers that Citi Bike uses in their published CSV data. For example some
+    # files use "Start Time", and others "starttime": both map to /begin/timestamp
+    projections:
+      bikeid: /bike_id
+      birth year: /birth_year
+      end station id: /end/station/id
+      end station latitude: /end/station/geo/latitude
+      end station longitude: /end/station/geo/longitude
+      end station name: /end/station/name
+      start station id: /begin/station/id
+      start station latitude: /begin/station/geo/latitude
+      start station longitude: /begin/station/geo/longitude
+      start station name: /begin/station/name
+      start time: /begin/timestamp
+      starttime: /begin/timestamp
+      stop time: /end/timestamp
+      stoptime: /end/timestamp
+      tripduration: /duration_seconds
+      usertype: /user_type
+
+materializations:
+  # Materialize rides into a PostgreSQL database.
+  examples/citi-bike/to-postgres:
     endpoint:
-      postgres: 
-        host: localhost 
-        password: flow 
-        user: flow
+      flowSink:
+        image: ghcr.io/estuary/materialize-postgres:dev
+        config:
+          # Try this by standing up a local PostgreSQL database.
+          # docker run --rm -e POSTGRES_PASSWORD=password -p 5432:5432 postgres -c log_statement=all
+          # (Use host: host.docker.internal when running Docker for Windows/Mac).
+          host: localhost
+          password: password
+          database: postgres
+          user: postgres
+          port: 5432
+    bindings:
+      # Flow creates a 'citi_rides' table for us and keeps it up to date.
+      - source: examples/citi-bike/rides
+        resource:
+          table: citi_rides
+
+storageMappings:
+  # Flow builds out data lakes for your collections in your cloud storage buckets.
+  # A storage mapping relates a prefix, like examples/citi-bike/, to a storage location.
+  # Here we tell Flow to store everything in one bucket.
+  "": { stores: [{ provider: S3, bucket: my-storage-bucket }] }
+```
+
+### Run It
 
+❗ These workflows are under active development and may change.
```

Comment:
We should probably add a note here saying that they need to go through the setup process per the docs in order for this to work (might not be the case for codespaces, but I imagine it is in at least some setups. It is the way I'm working). That doc is linked in the top of the readme so pointing them there would suffice.

Reply:
On a related note, it may be good to add codespaces setup to our current docs. I should also try it myself!